subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/SamplesDeclaringDependenciesIntegrationTest.groovy

```diff
@@ -43,6 +43,17 @@ class SamplesDeclaringDependenciesIntegrationTest extends AbstractIntegrationSpe
         sample.dir.file('build/libs/spring-web-5.0.2.RELEASE.jar').isFile()
     }
 
+    @UsesSample("userguide/dependencies/declaringBinaryDependenciesWithoutVersion")
```

Comment:
:x: Let's also change the terminology here as we don't want to call it "binary".

Reply:
Done